gateways/db/store/account/list_account_by_cpf_test.go

```diff
@@ -0,0 +1,80 @@
+package account
+
+import (
+	"context"
+	"errors"
+	"reflect"
+	"testing"
+
+	"github.com/Vivi3008/apiTestGolang/domain/entities/account"
+	"github.com/Vivi3008/apiTestGolang/gateways/db/store"
+)
+
+var SourceTest = "account_test.json"
```

Comment:
é comum colocarmos arquivos usados nos testes em pastas com nome testdata.
https://tutorialedge.net/golang/advanced-go-testing-tutorial/#use-the-testdata-directory